xlml/apis/task.py

```diff
@@ -77,6 +77,24 @@ def run(self) -> DAGNode:
 
     return group
 
+  def run_with_gcs_name_generation(self) -> DAGNode:
+    with TaskGroup(
+        group_id=self.task_test_config.benchmark_id, prefix_group_id=True
+    ) as group:
+      gcs_location = name_format.generate_gcs_file_location(
+          self.task_test_config.benchmark_id
+      )
+      provision, queued_resource, ssh_keys = self.provision()
+      run_model = self.run_model(
+          queued_resource, ssh_keys, {"gcs_location": gcs_location}
```

Comment:
ultra-nit: this variable becomes an env var, not a shell var, so please capitalize `GCS_LOCATION`

https://askubuntu.com/a/26322

Reply:
Now that I think about this, should we make this variable name more specific? We used to call this `MODEL_DIR` to match a convention in TF Estimator, but `OUTPUT_DIR` is clearer IMO